test/sample-data.json

```diff
@@ -208,9 +204,7 @@
                 },
                 {
                   "version_added": "49",
-                  "notes": [
-                    "A single note in an array"
-                  ]
+                  "notes": "A single note in an array"
```

Comment:
This makes no sense now

Reply:
That’s because I&nbsp;did a&nbsp;regex replace on&nbsp;the&nbsp;whole repository, and forgot to&nbsp;check for this.